### DISCUSSION TITLE
[FIX] Bumrad for Portal 2 is now working!

### DISCUSSION THREAD

**Discussion by mathieu12** [StrataSource/Portal-2-Community-Edition#1435] [Resources]
Have you encountered the "Can't create LogFile" error while compiling your map and using Bumrad?
That's normal; Portal 2 received an update that replaced some important files, which broke bumrad in the process.
Here's a fix I've made: this is the bin folder of an old Portal 2 build, with all the files from the previous version.
Just replace your actual bin folder with mine, and you're good to go :D

DOWNLOAD LINK: https://drive.google.com/file/d/1B_NAFkywxgYUn-_m-sBEE4v7VMYdzDlb/view?usp=sharing

If you have any questions or wanna thank me in person, search for "kazerlight" on steam or on the different discord servers :3

**Comment by JJL772** [comment]
~~Portal 2 Community Edition is a licensed Portal 2 mod, as such we don't use (or need) bumrad. I recommend posting this on a forum somewhere or perhaps in the TWP Discord server; portal 2 mappers aren't going to look here. Good work on the patch nonetheless, I'm sure portal 2 mappers will appreciate it!~~

EDIT: Didn't realize @ozxybox directed you here! I've added tags for use on the discussion board so we can differentiate between different resources, and which games they're compatible with. Thanks for posting!
